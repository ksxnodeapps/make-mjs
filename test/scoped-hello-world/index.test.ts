import message from '@scope-name/scoped-hello-world'

it('message matches snapshot', () => {
  expect(message).toMatchSnapshot()
})
