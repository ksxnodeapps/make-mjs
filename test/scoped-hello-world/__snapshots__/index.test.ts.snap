// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`message matches snapshot 1`] = `"Hello, World!!"`;
